app/v2/notifications/post_notifications.py

```diff
@@ -84,6 +84,13 @@
 )
 
 
+# TODO: replace with the real thing
+class RedisQueue:
```

Comment:
We opened that PR for you to work easily with an abstraction:
https://github.com/cds-snc/notification-api/pull/1437

Please review so we can work in tandem.